blog/_posts/2020-02-12-local-dev.md

```diff
@@ -0,0 +1,86 @@
+
+Expanding to support non-Kubernetes development environments gets us closer to our promise: if you use Kubernetes in Prod, you need Tilt for Dev.
+
+## What You Get
+
+After writing a Tiltfile, you'll have a better local development experience:
+* Start your whole app just by running `tilt up`
+* Update microservices automatically as you edit
+* Web UI that lets you see problems across services and zoom in on issues
+* Easily run workflows like resetting database state
+
+![Screenshot of Tilt with Local Dev](/assets/images/local-dev/screenshot.png)
+
+
+From this UI, you can see:
+* The list of servers and workflows ("Resources") in the sidebar on the right
+* All logs, multiplexed together
+* The status of each Resource (e.g. if a server died, its status will be red instead of green)
+* Logs for just one resource, by clicking on it in the sidebar
+
+## local_resource with serve_cmd
+Let's look at a sample Tiltfile for a project (simplified from our sample project [abc123](https://github.com/windmilleng/abc123/blob/master/Tiltfile.local)) that uses Tilt for two different cases:
+* a Go frontend that requires a build on each change
+* a Python backend ("numbers") that watches files and updates itself, except for a change to `requirements.txt`.
+
+```python
+local_resource(
+  'fe',
+  'go install github.com/windmilleng/abc123/fe',
+  serve_cmd='cd fe && fe --port=8000 --backend=localhost:8002',
+  deps=['fe'])
+
+local_resource(
+  'numbers',
+  'cd numbers && pip install -r requirements.txt',
+  serve_cmd='cd numbers && python app.py --port=%d' % (numbers_port),
+  # numbers is a flask app that handles most changes via hot reloading, but
+  # not requirements.txt changes
+  deps=['numbers/requirements.txt'])
+```
+
+This example project has clean separation between services. Tilt can handle more complex setups using options to `local_resource` that you can learn about in the [API reference](https://docs.tilt.dev/api.html#api.local_resource).
+
+## Ensuring consistency with `local_resource`
+If you're not using containers for development, you can lose some of the consistency of building an image from a Dockerfile. Adding a local_resource that checks local software installations can highlight incompatibilities before you spend hours debugging a build failure. Write a script called `check_compat.sh` that does the checks and add this to your Tiltfile:
+```python
+local_resource('check_compat', 'check_compat.sh')
+```
+
+Tilt will run this on startup and display an error if the command with an error.
+
+## Try Tilt With Your Project
+Download Tilt and start it. When you add to the Tiltfile, Tilt will automatically pick up the changes. If you want to pair program with a member of the Tilt team to configure Tilt for your local dev, you can [schedule a session](https://calendly.com/dbentley) or check out our [contact info](https://tilt.dev/contact).
```

Comment:
"When you add to the Tiltfile" feels weird. Maybe: "When you change your Tiltfile"?